specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2017-11-11-preview/searchindex.json

```diff
@@ -457,13 +463,19 @@
     },    
     "AutocompleteRequest": {
       "properties": {
+        "search": {
+          "type": "string",
+          "description": "The search text on which to base autocomplete results.",
+          "x-ms-client-name": "searchText"
+        },
         "autocompleteMode": {
           "$ref": "#/definitions/AutocompleteMode",
           "description": "Specifies the mode for Autocomplete. The default is 'oneTerm'. Use 'twoTerms' to get shingles and 'oneTermWithContext' to use the current context while producing auto-completed terms."
         },
         "fuzzy": {
           "type": "boolean",
-          "description": "A value indicating whether to use fuzzy matching for the autocomplete query. Default is false. when set to true, the query will autocomplete terms even if there's a substituted or missing character in the search text. While this provides a better experience in some scenarios it comes at a performance cost as fuzzy autocomplete queries are slower and consume more resources."
+          "description": "A value indicating whether to use fuzzy matching for the autocomplete query. Default is false. when set to true, the query will autocomplete terms even if there's a substituted or missing character in the search text. While this provides a better experience in some scenarios it comes at a performance cost as fuzzy autocomplete queries are slower and consume more resources.",
```

Comment:
nit: "Default is false. when" -> "Default is false. When"

Reply:
Fixed. Thanks!